server/routes/visitJourney/confirmation.ts

```diff
@@ -5,7 +5,7 @@ export default class Confirmation {
   constructor(private readonly mode: string) {}
 
   async get(req: Request, res: Response): Promise<void> {
-    const isUpdate = this.mode === 'update'
+    // const isUpdate = this.mode === 'update'
```

Comment:
Left commented out to avoid lint errors, as this will be needed when confirmation page takes account of booking vs update